app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -75,21 +73,37 @@ const MakeDecision = ({ data }) => {
     selectedItems: [],
     suggestedItems: [],
     startTime: Date.now(),
+    selectManualChoice: { issue: isBulkOperation ? { comment: '' } : itemData.issue },
+    suggestChoice: {},
+    historyChoice: {},
   });
-  const [tabs, toggleTab, collapseTabsExceptCurr] = useAccordionTabsState({
-    [MACHINE_LEARNING_SUGGESTIONS]: isAnalyzerAvailable,
-    [COPY_FROM_HISTORY_LINE]: false,
+  const [tabs, setActiveTab] = useState({
     [SELECT_DEFECT_MANUALLY]: true,
+    [MACHINE_LEARNING_SUGGESTIONS]: false,
+    [COPY_FROM_HISTORY_LINE]: false,
   });
```

Comment:
We can have only one active tab at a time, so there is no need to hold it as an object.